definitions/horizontal_scaling.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Horizontal Scaling
+status: Feedback Appreciated
+category: Concept
+---
+
+## Horizontal Scaling
+
+### What it is
+
+In [cloud computing](https://github.com/cncf/glossary/blob/main/definitions/cloud_computing.md) perspective Horizontal Scaling is type of [scalability](https://github.com/cncf/glossary/blob/main/definitions/scalability.md) whose approach is to add additional instances instead of moving into a larger instance size.
+
+For example, let us consider we have a system of 4GB RAM and we want to increase it's capacity to 16GB RAM, Horizontal Scaling means doing this by making a system of 4 X 4GB RAM rather than switching to a 16GB RAM system.
+
+It is an approach to enhance the performance of the server [node](https://github.com/cncf/glossary/blob/main/definitions/nodes.md) by adding new instances of the server to already existing servers in order to distribute the workload equally. In simple words it aims to decrease the server's load rather than expanding capacity of the individual server.
+
+Load balancing, clustering, and distributed file systems are some of the crucial and popular strategies of horizontal scaling. It caters to portioning of data where each [node]((https://github.com/cncf/glossary/blob/main/definitions/nodes.md)) contains single part of the data.
+
+### Problem it addresses
+
+With growing demands of the application the system should be able to scale by increasing/growing/managing the resources for the customers using it. The latency of the API calls might increase due to heavy traffic, this is the problem that can be solved by horizontal scaling.
```

Comment:
"increasing/growing/managing" we can't have the slashes in there. Ideally turn this into a couple complete sentences that convey the key problem horizontal scaling is meant to address. Something like: 

As demand for an application grows beyond the current capacity of that application instance an operator needs to find a way to add capacity by scaling. Either by adding more application instances or by giving more compute resources to the existing instances, or said another way, by scaling your application horizontally or vertically.